georates/main.py

```diff
@@ -48,7 +32,7 @@
 # Position (Lat, long)
 pos = df[[X, Y]].values.T
 # Values -> Thickness
-field = df[H].values.T
+field = field.T
 
 # Estimate the variogram
 bin_center, dir_vario = gs.vario_estimate(
```

Comment:
Need to specify the variogram fitting in two dimensions. That is, specify two angles for the variogram estimate and plot those two resulting variograms accordingly.